config/charon.yml

```diff
@@ -2,6 +2,15 @@ ignore_patterns:
   - ".*^(redhat).*"
   - ".*snapshot.*"
 
+artifact_suffix:
```

Comment:
I think we should use a reverse pattern to forbid the files of singing, because we don't know what's the valid types of a packages but we know what types are not needed to be signed.
Some files I think we should not sign as following:
general: .asc
maven: maven-metadata.xml, maven-metadata.xml.md5(.sha1,.sha256), archtype-catalog.xml, archtype-catalog.xml.md5(.sha1,.sha256)
npm: package.json

Reply:
Ok then

Reply:
Resolved, now uses ignore_artifact_suffix instead